driver.go

```diff
@@ -37,6 +37,8 @@ type Driver struct {
 	danglingKey    bool
 	ServerID       int
 	userData       string
+	volumeIDs      []string
```

Comment:
i would rename it to only `volumes`
```suggestion
      volumes        []string